addons/hr_timesheet/migrations/9.0.1.0/tests/test_hr_timesheet.py

```diff
@@ -0,0 +1,11 @@
+# coding: utf-8
+from openerp.tests.common import TransactionCase
+
+
+class TestHrTimesheet(TransactionCase):
+    def test_hr_timesheet(self):
+        self.env.cr.execute(
+            "SELECT line_id FROM hr_analytic_timesheet LIMIT 1")
+        line_id, = self.env.cr.fetchone()
+        line = self.env['account.analytic.line'].browse(line_id)
+        self.assertTrue(line.is_timesheet)
```

Comment:
my original idea was:

# you push the script
# notice that the test database doesn't cover all code
# push new commit adding the data in the database generation script
# push another commit that adds the test database generated with the new version of the script

The different commits allow us to squash the database commits so that they only live once in the commit history, in case we want to save space and bandwidth